internal/tool/files_by_language.go

```diff
@@ -0,0 +1,166 @@
+package tool
+
+import (
+	"io/fs"
+	"path/filepath"
+	"strings"
+)
+
+var filesByLanguage map[string][]string = make(map[string][]string)
```

Comment:
I don't think this needs to be global, right? It can be the return value of the methods below.

Reply:
It can, it then could be sent to the tool.go file as well as a parameter, but I thought that having this as a global instead of passing it as a parameter everywhere could make everything more readable.

Reply:
For me it's weird and error prone for this to be global. I see it as a product of running the tool.

If you run the tool twice, with different configurations, this the global variable will be re-used with incorrect values, since it is not cleared between runs.

Reply:
That is true, and I didn't think of it when implementing, however the way that the tool works is only on a one time run, and then the pod is terminated, so we would not get that error.